src/AppBundle/Entity/ClmAccount.php

```diff
@@ -50,14 +50,32 @@ class ClmAccount
      * User constructor.
      * @param string $name
      */
-    public function __construct($name)
+    public function __construct($name = '')
```

Comment:
EAdmin instantiates the object upon rendering the "new" form and thus cannot provide a name yet.
I could override the createNewClmAccountEntity() , but i would have to provide a dummy name there which then will be overwritten by the form input. Not very elegant either.
On the other hand $name is part of the primary key and thus required to persist the object to the database. I think its just the way easyadmin handles the entities.
Both ways will work. I'm eager for other suggestions from your side.